test/testdata/container_redis.json

```diff
@@ -3,7 +3,7 @@
 		"name": "podsandbox1-redis"
 	},
 	"image": {
-		"image": "quay.io/crio/redis:alpine"
+		"image": "redis:alpine"
```

Comment:
Why did you change this?

Reply:
I'm guessing a merge issue....  But should be reverted.